UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -954,7 +959,7 @@ private void SetConsciousState(ConsciousState NewConsciousState)
 		/// <param name="damageSplit">Should the damage be divided by number of body parts or applied to each body part separately</param>
 		[Server]
 		public void ApplyDamageAll(GameObject damagedBy, float damage, AttackType attackType, DamageType damageType,
-			bool damageSplit = true)
+			bool damageSplit = true, TraumaticDamageTypes traumaticDamageTypes = TraumaticDamageTypes.NONE, double traumaChance = 50)
```

Comment:
I thought the trauma damage was dependent on the attack type, 
e.g attack type sharp pointy thing = likelihood of getting bleeding, 
 what benefits from having the  chance of causing a trauma?

Reply:
It never worked that way, this has always been the way we detected traumas and their chances.

Reply:
You get to decide what each weapon and damage source can do in terms of trauma damage without being bound to existing damage types

Reply:
Can't you define that by attack type though?

Reply:
no, you can't.

Reply:
well The attack type Has blunt and sharp and stuff?

Reply:
You can only define burn via energy. But that itself is very limiting; what if you get burnt from other sources that aren't just energy? Traumas are better defined separately because everyone gets finer control over what traumas happen.

Reply:
> well The attack type Has blunt and sharp and stuff?

No, It doesn't. It only has "Brute" which is very hard to define in terms of slashing, piercing or blunt force traumas.

Reply:
Since we agreed that it's too limited with non-flag , To do is investigate flag attack types

Reply:
Also another good point is maybe We want a system where trauma isn't always present